src/java.desktop/share/classes/javax/swing/plaf/metal/MetalBorders.java

```diff
@@ -926,7 +925,7 @@ public static Border getTextFieldBorder() {
     }
 
     /**
-     * The class represents the border of a {@code JTestField}.
+     * The class represents the border of a {@code JTextField}.
```

Comment:
```suggestion
     * Border for a {@code JTextField}.
```
Such a description is used for other classes which aren't public.